experiment/cloud/secret_manager.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
```

Comment:
@inferno-chromium I want to start putting stuff in more specific modules than common.
Since there's no way this needs to be used at build time, it only needs to be used during an experiment (as with most cloud-related code, which I will move soon).

Reply:
Sounds good!